src/problems/Albany_Layouts.cpp

```diff
@@ -128,42 +128,42 @@ Albany::Layouts::Layouts (int worksetSize, int numVertices, int numNodes, int nu
   isSideLayouts = true;
 
   // Collapsed sideset layouts are now used to ensure contiguous memory access for efficient GPU evaluation
-  sidesetWorksetSize = singleWorksetSizeAllocation ? sidesetWorksetSize : worksetSize*numSides;
+  // sidesetWorksetSize = singleWorksetSizeAllocation ? sidesetWorksetSize : worksetSize*numSides;
```

Comment:
Remove unused inputs (e.g. worksetSize?) and comments.

Reply:
There are a few layouts that don't follow the <Side,...> pattern that use worksetSize as their first extent. (`node_node_scalar`, `node_node_vector`, and `node_node_tensor`)

I'm not using dynamic sizing for these at the moment since I don't know how they are used or if it is possible to reduce these allocations. Just a heads up as to why worksetSize is sticking around for the moment. The other unused inputs will be removed in my next commit.

Reply:
Can we try to remove them? What are they used for? We should try to use the <Side,...> everywhere.

Reply:
I queried the source and I didn't see any use for these. Let's go ahead and remove them and remove worksetsize unless you think there's another use for it.